docs/examples/layout/App/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+This component should act as the root of an application. In both environments, it will fill up the entire viewport and allow child components to flex appropriately.
```

Comment:
By both environments, do you mean both `native` and `web`?

Reply:
yea, maybe I don't need to specify at all since that is implied by the lack of tags?

Reply:
yep, i think so!